src/entities/IDealTypes.ts

```diff
@@ -22,7 +22,7 @@ export interface IDealVotingSummary {
 export interface IDealTokenSwapDocument {
   id: string;
   registrationData: IDealRegistrationTokenSwap;
-  discussions: any; // TODO
+  clauseDiscussions: Record<string, any>; // TODO should be IDealDiscussion, but breaks firebaseFunctions build
```

Comment:
Why does it break?  Can we fix that by moving the IDealDiscussion interface to where are located all the others that define the document?